changelogs/fragments/lookup_rabbitmq-is_closing-bug.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - lookup_rabbitmq - Fix for rabbitmq lookups failing when using pika v1.0.0 and newer.
```

Comment:
I don't know how lookup plugins are usually mentioned in changelog fragments, but wouldn't it be better to write something like `rabbitmq lookup plugin` (or `rabbitmq lookup`) instead of `lookup_rabbitmq`?

Reply:
Thanks @felixfontein - I've updated to your first recommendation.

Reply:
Updated as requested.